src/pyrqlite/cursors.py

```diff
@@ -58,25 +58,26 @@ def close(self):
 
 
     def _request(self, method, uri, body=None, headers={}):
-        debug = logging.getLogger().getEffectiveLevel() < logging.DEBUG
-        logging.debug(
+        logger = logging.getLogger(__name__)
```

Comment:
I think `logger` should be moved to outside the scope of any one function, in this source file. At least, that's how I've seen it done other times.